soql-standalone-parser/src/main/scala/com/socrata/soql/ast/Expression.scala

```diff
@@ -247,7 +247,9 @@ case class FunctionCall(functionName: FunctionName, parameters: Seq[Expression],
           sb <- parameters(1).format(d, sb, limit)
         } yield sb.append("]")
       case SpecialFunctions.StarFunc(f) =>
-        Some(sb.append(f).append("(*)"))
+        sb.append(f).append("(*)")
+        window.foreach(_.format(d, sb, limit))
```

Comment:
calling `format` can fail if the limit is non-None; you need to check its result (e.g., the way the Subscript function in the previous stanza does)